src/batou_scm/tests/test_buildout.py

```diff
@@ -1,10 +1,7 @@
-from batou.conftest import root
-from batou import UpdateNeeded
+efrom batou import UpdateNeeded
```

Comment:
```suggestion
from batou import UpdateNeeded
```

This line makes TravisCI unhappy.

Reply:
Ugh. Typo it seems. Not sure how it got in there as the tests did run for me ... -_-